source/localizable/_code-of-conduct.cz.md

```diff
@@ -0,0 +1,69 @@
+Účel
+-------
+
+Hlavním cílem všech konferencí a kommunit, které odkazují na tento Kodex chování, je zahrnout co největší počet osob s co nejrůznějším a nejrůznorodějším původem. Z toho důvodu se zavazujeme zajistit přátelské, bezpečné a přívětivé prostředí pro všechny bez ohledu na jejich pohlaví, sexuální orientaci, mentální nebo fyzický handicap, etnickou příslušnost, socioekonomický status a postoj k náboženství.
+
+Tento Kodex chování nastavuje očekávání od účastníků našich komunitních setkání a konferencí, a vyjmenovává důsledky v případě nepřijatelného chování.
+
+Žádáme všechny, kteří se účastní našich akcí, aby nám pomohli vytvořit bezpečné prostředí a pozitivní zážitky pro všechny.
+
+
+Pocit otevřeného občanství = Open [Source/Culture/Tech] Citizenship
+--------------------------------------
+
+Dodatečným cílem tohoto Kodexu chování je zvýšit pocit otevřeného občanství povzbuzováním účastníků k uznávání a posilování vazeb mezi jejich chováním a jeho dopady na naši komunitu.
+
+Komunity odrážejí společnost, ve které se nacházejí. Proto je nezbytný pozitivní vliv, aby došlo k vyrovnání se s mnoha formami nerovnosti a zneužívání moci, které ve společnosti existují.
+
+Prosím dejte nám vědět, pokud uvidíte někoho, kdo vynakládá větší úsilí, aby zajistil, že naše komunita je vstřícná, přátelská a povzbuzuje všechny účastníky.
+
+
+Očekávané chování
+-----------------
+
+* Účastněte se autentickým a aktivním způsobem. Tím přispějete ke zdraví a dlouhověkosti této komunity.
+* Při projevu a jednání s ostatními buďte ohleduplní a respektujte je včetně jejich názorů a postojů.
+* Snažte se o diskuzi, nevyvolávejte konflikt.
+* Zdržte se ponižujícího, diskriminačního nebo obtěžujícího chování a řeči.
+* Dbejte na své okolí a na ostatní účastnice a účastníky. Upozorněte organizátorky a organizátory setkání nebo konference, pokud si všimnete nebezpečné situace, někoho v nouzi nebo porušení tohoto Kodexu chování, i když se zdá, že jde o maličkost.
+
+
+Nepřijatelné chování
+---------------------
+
+Mezi nepřijatelné chování patří: zastrašující, obtěžující, urážlivé, diskriminační, hanlivé nebo ponižující projevy nebo činy kterékoliv účastnice a kteréhokoliv účastníka v komunitě online, offline, při všech souvisejících událostech a při individuální komunikaci prováděné v souvislosti s komunitou. Setkání komunity mohou být uskutečňována na místech, kde je přítomna veřejnost; respektujte všechny přítomné, i když nejsou součástí komunity.
+
+Mezi obtěžování patří: škodlivé nebo předpojaté slovní nebo písemné komentáře týkající se pohlaví, sexuální orientace, rasy, náboženství, zdravotního handicapu; nevhodné použití nahoty a / nebo sexuálních obrázků (včetně prezentačních snímků); nevhodné vyobrazení násilí (včetně prezentačních snímků); úmyslné zastrašování, pronásledování nebo následování; obtěžující fotografování nebo nahrávání; trvalé narušování rozhovorů nebo jiných událostí; nevhodný fyzický kontakt a nevítaná sexuální pozornost.
```

Comment:
This one is very contextual. I think "sledování" captures the meaning slightly better, however, it may be best to leave it out completely and use just "pronásledování".